style-guide.md

```diff
@@ -2,39 +2,39 @@
 
 > Style guide used in Kowainik.
 
-This document is a collection of the practices gained
-by working on the commercial and free open source Haskell libraries and applications.
+This document is a collection of best-practices inspired by commercial and free open source Haskell libraries and applications.
 
 ## Style guide goals
 
-The purpose of this document is to help developers and people working with
-Haskell sources to have smoother experience while dealing with code in different
-aspects. So, this style guide tries to provide these advantages by defining the
+The purpose of this document is to help developers and people working on
+Haskell code-bases to have a smoother experience while dealing with code in different
+situations. This style guide aims to increase productivity by defining the
 following goals:
 
-1. Make code **easier to understand:** the ideas for solutions should not be hidden
-   behind the complex and obscure code.
-2. Make code **easier to read:** code arrangement should be immediately apparent
-   after looking at the existing code.
+1. Make code **easier to understand:**  ideas for solutions should not be hidden
+   behind complex and obscure code.
+2. Make code **easier to read:** The flow of the code should be immediately apparent
```

Comment:
Here I wanted to say about code structure, alignment, position elements relative to each other and something like that. It's not architecture, because architecture is something high-level. But more about arrangement of code blocks. Does `the flow of the code` fits this description?

Reply:
Oh sorry I think I misunderstood it then. I was referring more to the execution paths of the code, which functions call which functions and the trail of execution until the end. 
I think you can  keep this as it was earlier.